Client/Frontend/Login Management/BreachAlertsManager.swift

```diff
@@ -30,50 +30,64 @@ final public class BreachAlertsManager {
     static let detailColor = UIColor(red: 0.59, green: 0.11, blue: 0.11, alpha: 1.00)
     static let monitorAboutUrl = URL(string: "https://monitor.firefox.com/about")
     var breaches = Set<BreachRecord>()
-    var breachAlertsClient: BreachAlertsClientProtocol
-
-    init(_ client: BreachAlertsClientProtocol = BreachAlertsClient()) {
-        self.breachAlertsClient = client
+    var client: BreachAlertsClientProtocol
+    var profile: Profile!
+    private lazy var cacheURL: URL = {
+        return URL(fileURLWithPath: (try? self.profile.files.getAndEnsureDirectory())!, isDirectory: true).appendingPathComponent("breaches.json")
```

Comment:
Would it be possible to remove the force unwrap here? If this needs to be an optional, that would probably be preferred.